test/config_parser_SUITE.erl

```diff
@@ -33,6 +33,21 @@ equivalence(Config) ->
     compare_unordered_lists(lists:filter(fun filter_config/1, Opts1), Opts2,
                             fun handle_config_option/2).
 
+miscellaneous(Config) ->
+    CfgPath = ejabberd_helper:data(Config, "miscellaneous.cfg"),
+    State1 = mongoose_config_parser_cfg:parse_file(CfgPath),
+    Hosts1 = mongoose_config_parser:state_to_host_opts(State1),
+    Opts1 = mongoose_config_parser:state_to_opts(State1),
+
+    TOMLPath = ejabberd_helper:data(Config, "miscellaneous.toml"),
+    State2 = mongoose_config_parser_toml:parse_file(TOMLPath),
+    Hosts2 = mongoose_config_parser:state_to_host_opts(State2),
+    Opts2 = mongoose_config_parser:state_to_opts(State2),
+
+    ?eq(Hosts1, Hosts2),
```

Comment:
I have mixed feelings about introducing shorter, less descriptive name for `?assertEqual`. Why not just use it?

Reply:
I was the one who introduced the `eq` macro here like we do for some other suites. For me if used very frequently, it makes the code a bit easier to read, that's it. What is more, I do add a debug printout to it from time to time ;-)

Reply:
I don't mind shorter names and `?eq` is pretty descriptive imo